doomed_patient/Guard.cpp

```diff
@@ -10,4 +11,12 @@ Guard::Guard(PatientGame* game, VectorXY startCoordinates)
 void Guard::move(Directions::Direction direction)
 {
 	centre += Directions::getDirectionVector(direction) * speed;
+
+	VectorXY playerPosition=game->player->getCentre();
+	double distance = sqrt(pow(centre.x - playerPosition.x, 2) + pow(centre.y - playerPosition.y, 2));
+	if (distance < 4)
+	{
+		
```

Comment:
I'd recommend putting a public method in player called `resetPosition()` that contains the following code:
`centre = Level::PLAYER_START.convertToWindowPosition()`

Then inside this if statement put:
`game->player->resetPosition()`

You might need to make sure you have `#include "Level.h"` in Player.cpp